### PR TITLE
embed shell scripts into nodelet go binary

### PR DESCRIPTION
**Pull request by roopakparikh**
This change include the whole of /opt/pf9/pf9-kube directory into the nodelet

### CODE REVIEW

File: nodelet/pkg/embedutil/pf9/pf9-kube/a.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+some random characters
```
Comment (pacharya-pf9) [inline nodelet/pkg/embedutil/pf9/pf9-kube/a.sh:1]: left over files from testing?

File: nodelet/pkg/pf9kube/embedkube.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+//go:embed etc/*
+var etc embed.FS
+
+func Extract() error {
+	zap.S().Infof("Extracting pf9-kube to '%s'", "/opt/pf9/")
+	efs := &embedutil.EmbedFS{Fs: kube, Root: "pf9"}
+	err := efs.Extract("/opt/pf9/")
+	if err != nil {
+		return fmt.Errorf("failed to extract pf9-kube: %s", err)
```
Comment (pacharya-pf9) [inline nodelet/pkg/pf9kube/embedkube.go:22]:
```suggestion
		return fmt.Errorf("failed to extract pf9-kube to /opt/pf9: %s", err)
```

File: nodelet/pkg/pf9kube/embedkube.go [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+	}
+	zap.S().Infof("Extracting etc to '%s'", "/etc/")
+	efs = &embedutil.EmbedFS{Fs: etc, Root: "etc"}
+	err = efs.Extract("/etc")
+	if err != nil {
+		return fmt.Errorf("failed to extract pf9-kube: %s", err)
```
Comment (pacharya-pf9) [inline nodelet/pkg/pf9kube/embedkube.go:28]:
```suggestion
		return fmt.Errorf("failed to extract pf9-kube to /etc: %s", err)
```

File: nodelet/pkg/embedutil/pf9/pf9-kube/a.sh (hunk 1 above)
Comment (roopakparikh) [reply nodelet/pkg/embedutil/pf9/pf9-kube/a.sh:1]: No this is for the "_test" file to test

File: nodelet/pkg/nodelet/cmd/initialize.go [hunk 4]
```diff
@@ -0,0 +1,26 @@
+package cmd
```
[the remark this answers is not in the document]
Comment (roopakparikh) [reply nodelet/pkg/nodelet/cmd/initialize.go:1]: A few reasons I added that:
a. With this change all I have is an independent init function - super useful for testing.
b. I was envisioning the rpm/deb to call init as post-install op - I have been afraid to touch that code so haven't implemented
c. I thought have init as a separate command may help when you want to make some changes to file without fear of overwriting whenever we are starting the nodelet

I am open to suggestion, I suggest we can take care of init to be called from the root if you prefer that way in a subsequent checkin

[the remark this answers is not in the document]
Comment (pacharya-pf9) [reply nodelet/pkg/nodelet/cmd/initialize.go:1]: Sounds good. Can you update the README for now?

[the remark this answers is not in the document]
Comment (roopakparikh) [reply nodelet/pkg/nodelet/cmd/initialize.go:1]: sure missed your comment, will add it.

**Review by pacharya-pf9 — APPROVED** [review]
LGTM

**Pull request merged**